pkg/apis/compliance/v1alpha1/scansettingbinding_types.go

```diff
@@ -76,6 +87,15 @@ func (s *ScanSettingBindingStatus) SetConditionReady() {
 	})
 }
 
+func (s *ScanSettingBindingStatus) SetConditionSuspended() {
+	s.Conditions.SetCondition(Condition{
+		Type:    "Suspended",
+		Status:  corev1.ConditionFalse,
+		Reason:  "Suspended",
+		Message: "The scan setting binding uses a scan setting that is suspended",
+	})
```

Comment:
I suppose this could include removing the `Ready` condition.

Reply:
Or we can use the Ready condition, set it to false, and then put `suspended` for the reason, which looks like it's more consistent with the rest of the controller.